### PR TITLE
1514 Unfriendly message when adding site with special characters in name

### PR DESCRIPTION
**Pull request by DaveBlueFrontier**

### CODE REVIEW

File: app/controllers/sites_controller.rb [hunk 1]
```diff
@@ -61,6 +62,12 @@ def create
     end
   end
 
+  def check_name
```
Comment (jorgegorka) [inline app/controllers/sites_controller.rb]: This should not be necessary. We need to enforce the use of UTF-8.  We already do on the database but we don't on the html. 

Adding:   
    %meta{:charset => "utf-8"}/   
to application.html.haml right after the %header should force browsers to use utf-8 and avoid problems

Comment (tripledub) [inline app/controllers/sites_controller.rb]: All the above, plus the controller is not the place to make this check even if we are going to force it

**Comment by tripledub** [comment]
How could we not have special characters in names? A clap-clinic in Thailand would be screwed? 
คลินิกตบมือ

**Comment by DaveBlueFrontier** [comment]
From my tests it doesn't seem to actually do anything anyway.  The site I created (from the example in the ticket) went through the creation process with no complaints or changes (it should have lost some characters in the process).  Though when I attempted the same site before the above change - it gave me an error.  

As far as the meta tag goes, that's a better way of doing it which is site-wide.

**Pull request merged**